development-docs/TESTING.md

```diff
@@ -391,6 +393,13 @@ To set the custom Kafka version in system tests, you need to set the Environment
 If you want to use private registries, before executing the tests, you have to create a secret and then specify the name of the created secret in the env variable called
 `SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET` with the container registry credentials to be able to pull images. Note that a secret has to be created in the `default` namespace.
 
+For tests that use `KafkaConnect` build feature, users can use their own registry to where the result image will be pushed.
+The config is specified by two env variables: `CONNECT_BUILD_IMAGE_PATH` and `CONNECT_BUILD_REGISTRY_SECRET`.
+`CONNECT_BUILD_IMAGE_PATH` has to be in format `<REGISTRY>/<ORGANIZATION>/<IMAGE_NAME>`. 
```

Comment:
I think it is worth mentioning explicitly that it should be without the tag and that the tag will be appended with the tests to ensure uniqueness.